packages/nuxt/src/core/runtime/nitro/renderer.ts

```diff
@@ -430,9 +430,13 @@ export default defineRenderHandler(async (event): Promise<Partial<RenderResponse
       style: []
     }
     for (const tag of await head.resolveTags()) {
-      if (tag.tag === 'link' && tag.props.rel === 'stylesheet' && tag.props.href.includes('scoped') && !tag.props.href.includes('pages/')) {
+      if (import.meta.dev && tag.tag === 'link' && tag.props.rel === 'stylesheet' && tag.props.href.includes('scoped') && !tag.props.href.includes('pages/')) {
```

Comment:
What would happen in production if an island use `useHead()` with a stylesheet <link> ? (can't test as i'm at work) Is it going to be ignored ?

Reply:
Yes, it would be ignored - but that wouldn't be inserted even now - it only inserts hrefs with `scoped` in their name which I believe is dev-only 🤔

I would be totally up for refactoring this so islands can insert more kinds of things in `<head>`, now that I've disabled the default resource hints from rendering there...

Reply:
oh i see that make sense :+1:

I think we could allow island to insert things into `head`. If this is too late for 3.9.2, this can probably be done in another PR ?

Reply:
good point - updated ✅